interactions/client/bot.py

```diff
@@ -126,7 +131,7 @@ def __register_events(self) -> None:
         self._websocket._dispatch.register(self.__raw_message_create, "on_message_create")
         self._websocket._dispatch.register(self.__raw_guild_create, "on_guild_create")
 
-    async def __compare_sync(self, data: dict, pool: List[dict]) -> bool:
+    async def __compare_sync(self, data: dict, pool: List[dict]) -> (bool, dict):
```

Comment:
Uhh, shouldn't that be `Tuple[bool, dict]`?
```suggestion
    async def __compare_sync(self, data: dict, pool: List[dict]) -> Tuple[bool, dict]:
```
May need to import it if not already